code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -311,6 +311,15 @@ obj/item/weapon/storage/box/syndie_kit/cheaptide
 	new /obj/item/weapon/reagent_containers/syringe(src)
 	return
 
+/obj/item/weapon/storage/box/syndie_kit/inverse_toxin
+	name = "box (C)"
+
+/obj/item/weapon/storage/box/syndie_kit/inverse_toxin/New()
+	..()
+	new /obj/item/weapon/reagent_containers/glass/bottle/protoxin(src)
+	new /obj/item/weapon/reagent_containers/syringe(src)
+	return
```

Comment:
no trailing return (even if it's consistent with the file)

Reply:
All removed.